CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ The `Unreleased` section name is replaced by the expected version of next releas
 
 ### Removed
  
+- Removed `netstandard2.1`; minimum target not `net6.0` [#407](https://github.com/jet/equinox/pull/407)
```

Comment:
```suggestion
- Removed `netstandard2.1`; minimum target now `net6.0` [#407](https://github.com/jet/equinox/pull/407)
```

Reply:
not that I could find

Reply:
My bad, seems I went aggressive on the doc update!